Wikipedia/Code/ArticleViewController.swift

```diff
@@ -786,6 +819,14 @@ private extension ArticleViewController {
         // Delegates
         webView.uiDelegate = self
         webView.navigationDelegate = self
+
+        webView.addSubview(refreshOverlay)
+        NSLayoutConstraint.activate([
+            refreshOverlay.topAnchor.constraint(equalTo: webView.topAnchor),
+            refreshOverlay.bottomAnchor.constraint(equalTo: webView.bottomAnchor),
+            refreshOverlay.leftAnchor.constraint(equalTo: webView.leftAnchor),
+            refreshOverlay.rightAnchor.constraint(equalTo: webView.rightAnchor),
+        ])
```

Comment:
There's a `wmf_addSubviewWithConstraintsToEdges` extension on `UIView` that might be helpful here

Reply:
👍🏽 thanks, will update.